CHANGELOG.md

```diff
@@ -9,7 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- `opentelemetry-instrumentation-redis` Add `sanitize_query` config option to allow query sanitization.  ([#1572](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1572))
+- Add default query sanitization for elasticsearch db.statement attribute
+  ([#1545](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1545))
```

Comment:
Your PR is not #1598 and
https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1598
?